controllers/controllers.go

```diff
@@ -34,14 +34,14 @@ var cookieStore = sessions.NewCookieStore([]byte("session-kesy"))
 
 // ここテストでstubするために関数ポインタをグローバル変数に代入しておきます．もしインスタンスメソッドではない関数をstubする方法があれば，書き換えて構わない．
 var CheckCSRFToken = checkCSRF
-var LoginRequired = checkLogin
+var LoginRequired = CheckLogin
 
 func CallController(controller interface{}, action string) interface{} {
 	method := reflect.ValueOf(controller).MethodByName(action)
 	return method.Interface()
 }
 
-func checkLogin(r *http.Request) (*userModel.UserStruct, error) {
+func CheckLogin(r *http.Request) (*userModel.UserStruct, error) {
```

Comment:
__[golint]__
* [golint] exported function CheckLogin should have comment or be unexported